src/main/java/edu/greenblitz/robotName/shootingStateService/ShootingPositionConstants.java

```diff
@@ -9,15 +9,7 @@ public class ShootingPositionConstants {
     
     public static final Translation2d CENTER_OF_SHOOTING_ZONE = FieldConstants.MIDDLE_OF_SPEAKER_POSITION.toTranslation2d();
 
-    public static final double OPTIMAL_SHOOTING_ZONE_RADIUS_IN_METERS = 2.5;
-
-    public static final double LEGAL_SHOOTING_ZONE_RADIUS_IN_METERS = 3;
-
-    public static ShootingZone OPTIMAL_SHOOTING_ZONE = new ShootingZone(
-            CENTER_OF_SHOOTING_ZONE,
-            OPTIMAL_SHOOTING_ZONE_RADIUS_IN_METERS,
-            FieldConstants.FieldZones.RESTRICTED_BOUNDS
-    );
+    public static final double LEGAL_SHOOTING_ZONE_RADIUS_IN_METERS = 5;
```

Comment:
```suggestion
    public static final double SHOOTING_ZONE_RADIUS_IN_METERS = 5;
```
it doesnt have to be the legal one by definition, it is just the one we chose.